bundler/src/main/java/com/f2prateek/bundler/FragmentBundleBuilder.java

```diff
@@ -0,0 +1,419 @@
+package com.f2prateek.bundler;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.util.SparseArray;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * A bundle builder for fragment arguments.
+ *
+ * @param <F> The fragment type you're bundling to.
+ */
+@SuppressWarnings("unchecked")
+class FragmentBundleBuilder<F extends Fragment> extends Bundler {
```

Comment:
1. How about just naming this `FragmentBundler`?
2. Does it need to extend `Bundler`? I think it makes more sense for this to wrap a `Bundler` instance internally that it simply delegates to.